doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.1.4
+- [Maintenance] Maintenance Commands are now handled with Symfony Messenger. The `pimcore:maintenance` command will trigger the maintenance messages to the bus. You should now also add `bin/console messenger:consume pimcore_core pimcore_maintenance` as a Cron Job
```

Comment:
Is it default behavior? or you should use `--async` option to use Symfony Messenger?